tests/common/test_zarr.py

```diff
@@ -1,12 +1,14 @@
 from pathlib import Path
 
+import pytest
 import zarr
 from pytest import MonkeyPatch
 
 from reformatters.common.config import Config, Env
 from reformatters.common.zarr import get_mode, get_zarr_store
 
 
+@pytest.mark.skip_set_test_final_store
 def test_get_zarr_store_dev(monkeypatch: MonkeyPatch) -> None:
     monkeypatch.setattr(Config, "env", Env.dev)
```

Comment:
This test really seemed like it should be the exception and not use our monkeypatching, since we want to assert things about the `dev` environment. I didn't find a better way to skip the `autouse` fixture besides this `pytest.mark` trick.